scripts/install-multi-user.sh

```diff
@@ -660,6 +780,17 @@ EOF
 }
 
 main() {
+    # TODO: I've moved this out of validate_starting_assumptions so we
+    # can fail faster in this case. Sourcing install-darwin... now runs
+    # `touch /` to detect Read-only root, but it could update times on
+    # pre-Catalina macOS if run as root user.
+    if [ $EUID -eq 0 ]; then
```

Comment:
```suggestion
    if [ $EUID == 0 ]; then
```

Reply:
I don't intend to modify this in the current PR--just move it. But, thanks for commenting--because looking at the diff made me realize that I forgot to delete it from the other location! <3